ios/FluentUI/Navigation/SearchBar.swift

```diff
@@ -144,6 +145,13 @@ open class SearchBar: UIView {
         static let navigationBarTransitionHidingDelay: TimeInterval = 0.5
 
         static let defaultStyle: Style = .lightContent
+        
```

Comment:
https://github.com/microsoft/fluentui-apple/runs/7220397515?check_suite_focus=true#step:5:147
Running pod lib lint should also catch these errors if your linter isn't

Reply:
cool, validated using`./scripts/validation.sh`